Amazon.IonObjectMapper.Test/Utils.cs

```diff
@@ -70,6 +70,17 @@ public static void Check<T>(IonSerializer ionSerializer, T item)
             Assert.AreEqual(item.ToString(), Serde(ionSerializer, item).ToString());
         }
 
+        public static void Check<T>(string expectedString, T item)
+        {
+            var serializer = new IonSerializer();
+            if (item == null)
+            {
+                Assert.AreEqual(null, Serde(serializer, (object)null));
```

Comment:
Shouldn't this still check the expectedString here?

Reply:
Hmm, actually why does this need to exist, shouldn't you simply call `item.ToString()` as in the method above?

Reply:
You are correct. If users are passing in `null` object they should be using the other function. This function is specifically for if the serde string representations are expected to not match.

Removed.